beacon-chain/powchain/service.go

```diff
@@ -297,6 +318,9 @@ func (w *Web3Service) ProcessChainStartLog(depositLog gethTypes.Log) {
 
 // run subscribes to all the services for the ETH1.0 chain.
 func (w *Web3Service) run(done <-chan struct{}) {
+	w.lastHeadInteractTime = time.Now()
```

Comment:
this line isnt wholly accurate, `w.lastHeadInteractTime` is not `time.Now()` , I think its better to set it only when we receive the block header from the ETH 1.0 chain below

Reply:
Interaction is both the new block and the subscribe itself. Just a subscription is an outgoing interaction. In this place i set lastHeadInteractTime because call `w.reader.SubscribeNewHead`

Reply:
in that case could you put the line below  `w.reader.SubscribeNewHead`

Reply:
I removed the lastHeadInteractTime